src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="/"
+      class="search wrap-big"
+      data-qa="big"
+    >
+      <img
+        class="search__icon search__icon--big"
+        src="./images/Search.svg"
+        alt="search">
```

Comment:
```suggestion
      <img
        class="search__icon search__icon--big"
        src="./images/Search.svg"
        alt="search"
      >
```
Fix all cases
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96409563/7d8e48e7-fed6-4929-8ec7-e34e648aff87)